libs/aml/src/amlParser.ts

```
@@ -62,53 +64,54 @@
 import {badIndent, legacy} from "./errors";
 
 // special
-const WhiteSpace = createToken({name: 'WhiteSpace', pattern: /[ \t]+/})
-const Identifier = createToken({ name: 'Identifier', pattern: /\b[a-zA-Z_][a-zA-Z0-9_#]*\b|"([^\\"]|\\\\|\\"|\\n)*"/ })
-const Expression = createToken({ name: 'Expression', pattern: /`[^`]+`/ })
+const Comment = createToken({ name: 'Comment', pattern: /#[^\n]*/ })
 const Doc = createToken({ name: 'Doc', pattern: /\|(\s+"([^\\"]|\\\\|\\")*"|([^ ]#|[^#\n])*)/ }) // # is included in doc if not preceded by a space
 const DocMultiline = createToken({ name: 'DocMultiline', pattern: /\|\|\|[^]*?\|\|\|/, line_breaks: true })
```

Biome

[error] The regular expression includes this negated empty character class.

Negated empty character classes match anything.
If you want to match against [, escape it \[.
Otherwise, remove the character class or fill it.

(lint/correctness/noEmptyCharacterClassInRegex)